src/api/handlers/auth/authenticate.js

```diff
@@ -0,0 +1,31 @@
+import jwt from "jsonwebtoken";
+import { jwtConfig } from "../../../config";
+
+/*
+ * Validate a users auth token
+ */
+export function authenticate(req, res, next) {
+  // check header or url parameters or post parameters for token
+  let token = req.headers["x-access-token"] || req.headers.authorization;
+  if (!token) return res.json({ success: false, message: "Unauthorized." });
```

Comment:
Does res.json default to 200?

Reply:
Yes. This will return 200 Unauthorized. :rofl: I will update, good catch. Coding at :night_with_stars: :man_shrugging:

Reply:
Yes, `res.status(401).json({ message: "Unauthorized." })` should do it.